enterprise/admin/views.py

```diff
@@ -241,7 +242,7 @@ def get_search_keyword(self, request):
         """
         return request.GET.get('q', None)
 
-    def get_enterprise_customer_user_queryset(self, search_keyword, customer_uuid):
+    def get_enterprise_customer_user_queryset(self, request, search_keyword, customer_uuid):
```

Comment:
nit: please add `request` in `Args` below and also rename `Args` to complete word `Arguments` http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings